.buildkite/pipeline.yml

```
@@ -7,24 +7,23 @@ env:
     && buildkite-agent artifact download stablecoin.tz haskell/test/resources/
     && buildkite-agent artifact download stablecoin.fa1.2.tz haskell/test/resources/
     && buildkite-agent artifact download metadata.tz haskell/test/resources/ --step "LIGO-contract"
-    && cp "./nix/sources.json" "./haskell/test/resources/"
   TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER: "Y"
 
 .network-tests: &network-tests
   commands:
     - eval "$FETCH_CONTRACT"
     - cd haskell/
-    - export PATH=$(nix-build ../ -A stablecoin-client)/bin:$PATH
-    - nix build -L -f .. test
+    - export PATH=$(nix build ../ .#stablecoin:exe:stablecoin-client)/bin:$PATH
```

```suggestion
    - export PATH=$(nix build ../ .#stablecoin:exe:stablecoin-client --print-out-paths)/bin:$PATH
```
Same for every such usage

---

I would also provide `--no-link`